apps/omg_child_chain/lib/omg_child_chain/datadog_event/contract_event_consumer.ex

```diff
@@ -84,9 +84,11 @@ defmodule OMG.ChildChain.DatadogEvent.ContractEventConsumer do
 
     options = tags(aggregation_key, state.release, state.current_version, timestamp)
     title = "#{event_name}"
-    message = "#{inspect(Encode.make_it_readable!(data))} - Timestamp: #{timestamp}"
 
-    :ok = apply(state.publisher, :event, create_event_data(title, message, options))
+    Enum.each(data, fn ev ->
+      message = "[#{inspect(Encode.make_it_readable!(ev))}] - Timestamp: #{timestamp}"
+      :ok = apply(state.publisher, :event, create_event_data(title, message, options))
```

Comment:
The `title` is derived from `hd(data)`, so it's assuming the first event name is the same as the rest.

In this PR where we're already breaking down per event, should we derive it from each event instead?

Reply:
True, that would make more sense. Is there any event other than the `BlockSubmitted` which could come in bulk?

Reply:
I didn't look into specifics but I think `DepositCreated` and exit started events. Need double checking though, I havn't touched on them lately.

Reply:
And `ExitFinalized`, some others as well